plugins/modules/pacman_key.py

```diff
@@ -78,9 +78,9 @@
     default: /etc/pacman.d/gnupg
   state:
     description:
-      - Ensures that the key is present (added) or absent (revoked).
+      - Ensures that the key is present (added), trusted (signed and not expired) or absent (revoked).
```

Comment:
You should mention when `trusted` was added:
```suggestion
      - Ensures that the key is V(present) (added), V(trusted) (signed and not expired) or V(absent) (revoked).
      - The state V(trusted) has been added in community.general 10.6.0.
```
Also formatting the state names as values is IMO a good idea.

Reply:
If you're not opposed to folding the new functionality into the `present` state, I think that would be preferable. IMHO the currently implemented functionality of the `present` state does not solve a real life use case.

Reply:
I think this would be a breaking change, since this considerably changes the behavior of `state=present`. Since we don't accept breaking changes this isn't really acceptable.

(That's why I would solve this not with a new state, but with an option. That way we can deprecate its default, and eventually change it to the better behavior, and fix the issue that way.)

Reply:
Thanks! Finally got around to making the change to make it an option now.

I squashed all commits now, but I could restore the individual commits as well of course. Just let me know what's easier to review. Not super familiar with GitHub review workflows.